modules/running-distributed-data-science-workloads-disconnected-env.adoc

```diff
@@ -0,0 +1,48 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-disconnected-env_{context}"]
+= Running distributed data science workloads in a disconnected environment
+
+[role='_abstract']
+To run a distributed data science workload in a disconnected environment, you must have access to a Ray cluster image, and to the data sets and Python dependencies used by the workload tasks.
+
+.Prerequisites
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+* You have access to the disconnected data science cluster.
+* You have installed {productname-long} and created a mirror image as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_openshift_ai_self-managed_in_a_disconnected_environment[Installing and uninstalling {productname-short} Self-Managed in a disconnected environment].
+* You have access to a Ray cluster image.
+* You have access to the data sets and models to be used by the workload.
+* You have access to the Python dependencies for the workload, either in a Ray image or in your own Python Package Index (PyPI) server that is available from the disconnected cluster.
+* You have logged in to {productname-long}.
+* You have created a data science project.
+
+.Procedure
+. Configure the disconnected data science cluster to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+. Edit the notebook or pipeline to provide the location of the Ray cluster image:
```

Comment:
Just a note - Ray cluster image has to be provided by user in all cases, even for "normal" deployment.
So, maybe something like "Make sure that Ray cluster image referenced in notebook or pipeline points to image available in disconnected environment."?